HashMap.py

```diff
@@ -0,0 +1,51 @@
+// Time Complexity :O(1)
+// Space Complexity :
```

Comment:
please add the space complexity

Reply:
Try to solve this problem using approach taught in class, it's more efficient w.r.t time and space